### PR TITLE
added business stakeholder endpoints

### PR DESCRIPTION
**Pull request by madhankumar3103**

### CODE REVIEW

File: src/Hyperwallet.js
Comment (arao6) [inline src/Hyperwallet.js:1821]: Please add a blank line here

File: src/Hyperwallet.js
Comment (arao6) [inline src/Hyperwallet.js:1849]: Please change this method name to - listBusinessStakeholders

File: src/Hyperwallet.js
Comment (arao6) [inline src/Hyperwallet.js:1874]: you can call 'createBusinessStakeholderStatusTransition' method, without making an API call from here

File: src/Hyperwallet.js
Comment (arao6) [inline src/Hyperwallet.js:1894]: you can call 'createBusinessStakeholderStatusTransition' method, without making an API call from here

File: src/Hyperwallet.js
Comment (arao6) [inline src/Hyperwallet.js:1914]: Please add a blank line

File: src/Hyperwallet.js
Comment (arao6) [inline src/Hyperwallet.js:1894]: Could you change this to call the createBusinessStakeholderStatusTransition, instead of making an API call from here.

File: src/Hyperwallet.js
Comment (arao6) [inline src/Hyperwallet.js:1875]: Could you change this to call the createBusinessStakeholderStatusTransition, instead of making an API call from here.

File: src/Hyperwallet.js
Comment (arao6) [inline src/Hyperwallet.js:1949]: please change this method name to 'listBusinessStakeholderStatusTransitions'

File: src/Hyperwallet.js
Comment (arao6) [inline src/Hyperwallet.js:1958]: blank line here pls

File: test/Hyperwallet.spec.js
Comment (arao6) [inline test/Hyperwallet.spec.js:4308]: please add blank lines between the tests

File: test/Hyperwallet.spec.js
Comment (arao6) [inline test/Hyperwallet.spec.js:4539]: blank line here pls

File: test/Hyperwallet.spec.js
Comment (arao6) [inline test/Hyperwallet.spec.js:4649]: blank line

File: test/Hyperwallet.spec.js
Comment (arao6) [inline test/Hyperwallet.spec.js:4664]: blank lines between the it tests

**Review by arao6 — APPROVED** [review]

**Review by vkorotchenko — APPROVED** [review]

**Review by Rrathinasabapath — APPROVED** [review]
Verified changes. Changes can be merged after approval by Vadim.

**Pull request merged**
